usecases/config/environment.go

```diff
@@ -444,6 +444,22 @@ func FromEnv(config *Config) error {
 		return fmt.Errorf("parse sentry config from env: %w", err)
 	}
 
+	config.MetadataServer.Enabled = false
+	if entcfg.Enabled(os.Getenv("EXPERIMENTAL_METADATA_SERVER_ENABLED")) {
```

Comment:
(not a blocker. can be a follow up PR :) )

I think better to have a cli-flag for this config as well?
`experimental.metadata-server.enabled` or something similar. 

Rationale are.
1. ENVs can easily get messy at scale (10s of ENV is fine. 100s will be crazy). I want my configs to be "local" and "explicit". ENVs are by definition "global" and "implicit".
2. I don't mind having envs, but I think we should have priorities in order,  cli-flags > envs. Basically, [this comment is no longer true unfortunately in handling configs](https://github.com/weaviate/weaviate/blob/main/usecases/config/config_handler.go#L419-L424). I would love to keep it true for things at least we add from serverless sides of things :)